pyproject.toml

```diff
@@ -47,6 +47,8 @@ source = [
   "staff",
   "tests",
 ]
+# Required to get full coverage when using Playwright
```

Comment:
I might have misunderstood what this is doing, but do we want our playwright tests to be included in coverage? Should we be relying on our other tests to get sufficient coverage instead?